src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
```

Comment:
I don't see where you import this file. And you already add styles for stars in card.scss. Please delete it